### PR TITLE
fix option name

### PR DESCRIPTION
**Pull request by mairaw**
LiveFyre comment for topic https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet:

"The --verbose option is pretty confusing. 

First, the .NET Core SDK 2.0.2 reports the option as named --verbosity (not --verbose) and it requires a value from a set composed by q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].

Then, I tried every possible combination but I still can't understand how they're supposed to work. None of these commands succeeded (tried with dotnet new console).

dotnet new console -v
dotnet new console -v d
dotnet new console -vd
dotnet new console -v=d
dotnet new console --verbose

Please improve the documentation."

@dsplaisted @livarcocc does dotnet new support the verbosity flag? This is still listed as a common option when we do `dotnet --h` but it doesn't seem to work for all commands.

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/core/tools/dotnet?branch=pr-en-us-3624&tabs=netcore2x)

### CODE REVIEW

**Review by dsplaisted — APPROVED** [review]

File: docs/core/tools/dotnet.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Prints out detailed information about the CLI tooling and the environment, such
 
  Rolls forward on no candidate shared framework.
 
-`-v|--verbose`
+`-v|--verbosity`
 
 Enables verbose output.
```
Comment (dsplaisted) [inline docs/core/tools/dotnet.md]: This should specify the options for verbosity, which I think are: `q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic]`

**Review by jzeferino — APPROVED** [review]
This LGTM.

**Review by BillWagner — APPROVED** [review]
I ran through the commands here, and this LTGM. 

@mairaw you can :shipit: when ready

**Comment by mairaw** [comment]
Sorry for the delay here. I made the suggested changes but I'm still not sure whether it's the right thing to list this as a common option or if we should simply remove this option from this page since it doesn't work with every command. Thoughts @dsplaisted?

**Review by KathleenDollard — APPROVED** [review]

**Comment by KathleenDollard** [comment]
Can we just mark it with a note? It clarifies that it should always be just like this if it exists, but it doesn't always exist.

**Comment by mairaw** [comment]
Sure, I'll add that @KathleenDollard.

**Comment by mairaw** [comment]
@dsplaisted I decided to run some final tests before merging this. it seems that although dotnet --help for 1.x versions lists the option name as --verbose, all the commands never accepted that and in fact just dotnet --help was displaying the wrong information for 1.x. Can you confirm? 
I'll be adjusting this PR to reflect what I'm seeing on my tests.

/cc @KathleenDollard

**Comment by dsplaisted** [comment]
@mairaw I believe the help text was incorrect for 1.x, and we fixed it in 2.0.  See https://github.com/dotnet/cli/issues/5502 and https://github.com/dotnet/cli/pull/6690

**Comment by mairaw** [comment]
Thanks @dsplaisted. So merged this with my last fix.

**Pull request merged**